plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/UnsupportedGradleTest.kt

```diff
@@ -14,6 +16,7 @@ class UnsupportedGradleTest : AbstractPluginTest() {
             .withGradleVersion("5.6.4")
 
     @Test
+    @DisabledOnOs(OS.WINDOWS)
```

Comment:
What's up here?

Reply:
Gradle test daemon disappearing on Github Windows runner. I've tried to reproduce it on another windows installation, but test there is passing successfully.

And actually it is ok to run this test only on Linux runner.